quil-rs/src/program/graph.rs

```diff
@@ -528,6 +530,7 @@ fn terminate_working_block(
     let block = InstructionBlock::build(std::mem::take(working_instructions), terminator, program)?;
     let label = working_label
         .take()
+        .cloned()
         .unwrap_or_else(|| ScheduledProgram::generate_autoincremented_label(blocks));
```

Comment:
`working_label` changed from `Option<String>` to `Option<&String>`, but the `unwrap_or_else` must return a `String`. So, we must clone the string if `working_label.is_some()`